src/styles.css

```diff
@@ -0,0 +1,54 @@
+h1{
+  font-size: 80px;
+  font-family: 'Bebas Neue', cursive;
```

Comment:
I'd love to see a screenshot of the app in the PR next time you submit one. It helps me put some context and see the great design you're working on.

Reply:
Here's a quick screenshot of the todo list feature and how it has been designed so far.

<img width="960" alt="CheckItOff1" src="https://user-images.githubusercontent.com/23369060/213746210-951b45f8-7918-420e-a3d9-49c58175ffb0.PNG">